modules/apps/portal-search/portal-search-api/src/main/resources/content/Language.properties

```diff
@@ -18,6 +19,7 @@ indexing-batch-sizes=Indexing Batch Sizes
 indexing-batch-sizes-help=The number of documents to index per batch for model types that support batch indexing. Defaults to 10000. For models with large documents, decreasing this value may improve stability when executing a full reindex.
 max-buffer-size=Maximum Buffer Size
 max-expansions=Maximum Expansions
+max-expansions-help=The number of unique titles that can be matched when search terms match a title as a phrase prefix.
```

Comment:
Not entirely sure I understand this one. Let me know if this distorts the intended meaning:

> Specify the maximum number of unique titles to return when the searched keywords match a title as a phrase prefix.

Reply:
ya, its complicated. again can we just add "Set..." to the beginning of the sentence?

Reply:
Sure.